### PR TITLE
Feat/local storage

### LINKED ISSUE #23: LocalStorage
## Value Statement
As a user, I want to save my trips to the app in order to see them anytime I want, change them or remove them from the app.

## Acceptance Criteria
- [x] save trip to localStorage
- [x] edit trip in localStorage
- [x] remove trip from localStorage

## Tasks
- [x] create function to add trips to localStorage
- [x] create function to get a single trip from localStorage
- [x] create function to edit a single trip
- [x] create function to remove a single trip

## Size
large

### PR DESCRIPTION
**Pull request by yvlange**
- save the input of the text areas in the localStorage
- created a new services folder and placed the localStorage functions in that folder to organise the code

### CODE REVIEW

File: src/components/AddForm.js [hunk 1]
```diff
@@ -11,10 +15,18 @@ function AddForm() {
...
-    console.log(restaurantsInput);
-    console.log(notesInput);
+    addTripsToLocalStorage({
+      destination: destinationInput,
+      activities: activitiesInput,
+      restaurants: restaurantsInput,
+      notes: notesInput,
+    });
+    const trips = getTripsFromLocalStorage();
+    console.log(trips);
```
Comment (FriederikeM) [inline src/components/AddForm.js:25]: console log detected 👀

**Review by FriederikeM — APPROVED** [review]
To me this all looks very good. Well done getting local storage under control. I found just one console log you left in ;)

File: src/components/AddForm.js (hunk 1 above)
Comment (yvlange) [reply src/components/AddForm.js:25]: @FriederikeM You are my "Adlerauge" of the console log! 🙏

**Review by soerendolling — APPROVED** [review]
Yay local storage. Nice work

**Pull request merged**